packages/components/src/internal/components/domainproperties/actions.ts

```diff
@@ -401,6 +401,11 @@ export function saveDomain(options: SaveDomainOptions): Promise<DomainDesign> {
                 reject(response);
             }
 
+            // Issue 49113: Better handling for case where the only error is a form error
+            if (response.errors.length === 1 && response.errors[0].id === 'form') {
+                reject(response);
```

Comment:
resolveErrorMessage?

Reply:
the usages of `resolveErrorMessage` is in the `catch` block of the ListDesignerPanels.tsx component